modules/special-rules/pages/war-engines-and-blast-markers.adoc

```diff
@@ -1,11 +1,29 @@
 = War engines and Blast markers
 
-Blast markers affect War Engine detachments in much the same way as other units. 
+Blast markers affect War Engine detachments in much the same way as other units.
 However, *all* War Engine detachments have the {stubborn} ability.
 
-Note that for a War Engine detachment has a total initial Damage Capacity value of more than `15`, you may add +1 to the result of the Leadership test.
+You may add +1 to the result of Leadership tests when the total initial Damage Capacity value of the War Engine detachment is `15` or more, instead when it has `15` or more units.
+
 {blank}[TODO: Is this the best place to highlight this point? I think it might be better to improve the Leadership test text and to clarify what is what when it comes to 'number of units' and 'detachment strength' and initial Damage Capacity versus current Damage Capacity for the purposes of the Leadership test.]
```

Comment:
I'm still not sure about this. It'd be better on the one hand to refine the Leadership test topic and eliminate the need to mention it here — but on the other hand it would mean that we'd necessarily have to mention some special rules concepts, which we try not to do in the core rules.

@IJW-Wartrader Do you have any more thoughts on the matter, either way? If not then we could leave this note as-is for now and experiment with the idea another time in future — perhaps when we're getting nearer to a first stable release and we go back over the rules to clear up all TODO notes in one way for another.

Reply:
Actually, given your addition of examples below, I think it makes most sense to do it all as we've done here — otherwise we kinda need to add the examples to the core rules as well. So, might as well delete this note I think.
```suggestion
```